alpha-features.md

```diff
@@ -0,0 +1,28 @@
+# Open Service Broker API Alpha Features
+
+## Table of Contents
+  - [Alpha Overview](#overview)
+  - [How alpha features are documented](#how-alpha-features-are-documented)
+  - [Current alpha features](#current-alpha-features)
+
+## Overview
+
+The features described in this document are released at an ALPHA level.  They
+are are available for use in order for the community to provide feedback and
+are not yet an official part of the spec.
+
+Alpha features may be changed or removed in response to feedback.
+
+## How Alpha features are documented
+
```

Comment:
I know this came up on the call, but just to jog my memory. What are the downsides of adding new fields or sections to the spec with a big "ALPHA" label?